Modules/HEART_CLASS/heart_class.h

```diff
@@ -0,0 +1,82 @@
+#ifndef DADM_HEART_CLASS_H
+#define DADM_HEART_CLASS_H
+
+#include <complex>
```

Comment:
Wszystkie `#include` które możemy, przenosimy do pliku `.cpp`. Zoptymalizuje to proces kompilacji kodu.